client/components/signup-form/index.jsx

```diff
@@ -67,7 +90,11 @@ export default React.createClass( {
 			hideFieldErrorsOnChange: true,
 			initialState: this.props.step ? this.props.step.form : undefined
 		} );
-		this.setState( { form: this.formStateController.getInitialState() } );
+		let initialState = this.formStateController.getInitialState();
+		if ( abtest( 'autoFillUsernameSignup' ) === 'autoFill' ) {
```

Comment:
`SignupForm` is mounted in the main signup at `/start` in the `user` step, but also elsewhere in Calypso when users accept an invite, in `LoggedOutInvite`. We only have the progress store in the former case, so we probably only want to assign them to the test if that is present. We can achieve this by checking for the `signupProgressStore` here:

```js
if ( this.props.signupProgressStore && abtest( 'autoFillUsernameSignup' ) === 'autoFill' ) {
```

This will also allow you to remove the checks for `! signupProgressStore` inside the utility functions.